src/ckill.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+# fsid
+if [ -e fsid ] ; then
+    fsid=`cat fsid`
+else
+    fsid=`uuidgen`
```

Comment:
you can't just generate a new fsid to kill an existing cluster, right?

Reply:
if you say ``rm fsid`` you'll just start up a new cephadm cluster.  the old one needs to be removed by running ``ckill.sh`` first or running ``cephadm rm-cluster --fsid ... --force``.

Reply:
I understand; I'm just saying, here in ckill, generating a new fsid makes no sense.  You want to find the fsid of the cluster to be killed, surely?

Reply:
oh i see, yeah.  fixed! :man_facepalming: